Generals/Code/GameEngine/Source/Common/System/AsciiString.cpp

```diff
@@ -131,7 +131,8 @@ void AsciiString::ensureUniqueBufferOfSize(int numCharsNeeded, Bool preserveData
 	{
 		// no buffer manhandling is needed (it's already large enough, and unique to us)
 		if (strToCopy)
-			strcpy(m_data->peek(), strToCopy);
+			// TheSuperHackers @fix Mauller 04/04/2025 ASAN, Replace strcpy with safer memmove as copy occurs within overlapping address space
```

Comment:
No need to mention asan here.

"Replaces strcpy with memmove because copy can occur within overlapping address space"